include/OsmAndCore/PointsAndAreas.h

```diff
@@ -160,6 +160,11 @@ namespace OsmAnd
         {
             return static_cast<T>(qSqrt(squareNorm()));
         }
+
+        inline PointT normalized() const
+        {
+            return (*this/norm());
```

Comment:
why outer () are needed? Also, for some odd reason primary operators are space-divided elsewhere:
`a * b` but `a*b + c*d`

Reply:
Regarding last - no strong opinion, there's no official codestyle

Reply:
There are no any specific reason. Sometime i used this style hen there are no any strict code style.
Some explanation: https://stackoverflow.com/a/161899/9968718